docs/components/copy.md

```diff
@@ -0,0 +1,164 @@
+# Copy
+
+**KCopy** is a component that displays text content and copies it to clipboard.
+
+<KCopy :content="content" />
+
+```html
+<KCopy :content="content" />
+```
+
+## Props
+
+### badgeText
+
+Text displayed before the copyable content when `isBadge` is true.
+
+<KCopy
+  isBadge
+  badge-text="Id"
+  :content="content"
+/>
+
+```html
+<KCopy
+  isBadge
+  badge-text="Id"
+  :content="content"
+/>
+```
+
+### content
+
+The copyable text.
+
+### contentTooltip
+
+Tooltip text displayed on hover over the `content`.
+
+<KCopy
+  truncate
+  :content="content"
+  content-tooltip="Hello!"
+/>
+
+```html
+<KCopy
+  truncate
+  :content="content"
+  content-tooltip="Hello!"
+/>
+```
+
+### truncate
+
+Whether or not the content should be truncated.
+> Note: By default truncate is set to false.
+
+<KCopy
+  truncate
+  :content="content"
+/>
+
+```html
+<KCopy
+  truncate
+  :content="content"
+/>
+```
+
+### truncationLimit
+
+Number of characters to truncate at.
+> Note: By default it is set to `8`.
+
+<KCopy
+  truncate
+  :content="content"
+  truncation-limit=10
+/>
+
+```html
+<KCopy
+  truncate
+  :content="content"
+  truncation-limit=10
+/>
+```
+
+### copyTooltip
+
+Tooltip text displayed on hover copy button. 
+> Note: If `badgeText` exists then text is `Copy <badge-text>` else `Copy`
```

Comment:
```suggestion
Tooltip text displayed on hover copy button. 

If the `badgeText` prop has a value, then the copy tooltip text is `Copy {badgeText}`; otherwise, `Copy`.
```